.github/README.md

```diff
@@ -17,6 +20,21 @@ Welcome to **Backdrop**, a command-line utility designed to manage your desktop
 
 ## :wrench: Installation
 
+you can install Backdrop by downloading and unziping the `tar.gz` file for your operating system (if available) that is included in the release and place it under the following path:
+```bash
+mkdir -p $HOME/.backdrop/bin
+cp backdrop $HOME/.backdrop/bin
+```
+
+or you can install it by compiling the source code
+```bash
+git clone https://github.com/JanMichaelSE/backdrop.git
+cd backdrop
+go build
+go install
+```
+
+<!--NOTE: not sure if we should leave this portion on -->
```

Comment:
Yes we do need this and it should be the first option. Since one can install that app the easiest this way. But I did found out that `go get` changed in newer versions. And now you only need to do the `go install` one.